### PR TITLE
Bump to building state with current autotools and gcc

### PR DESCRIPTION
**Pull request by onkobu**
- bump version
- configure.ac instead of .in
- tagging of libtool (CXX)
- fixed constexpr
- fixed template syntax

### CODE REVIEW

File: LEAME [hunk 1]
```diff
@@ -1,207 +1,179 @@
...
-Autor :
--------
-Pueden contactarme (Stefan Hetzl) por e-mail: shetzl@chello.at
-
-
-Traducción al castellano :
---------------------------
-versión 1 - Alberto Adrián Schiano (chanio@users.sourceforge.net)
-
+Introducci�n :
```
Comment (StefanoDeVuono) [inline LEAME:1]: Can you check the encoding here?

Comment (onkobu) [reply LEAME]: Not only checked but also fixed. Somehow this turned into ISO-8859-1. Reverted it back to UTF-8.

**Comment by Aminuxer** [comment]
Does need some fixes in russian translation ?

**Comment by Aminuxer** [comment]
make new version of RU-translation, pull request sended.

File: po/LINGUAS [hunk 2]
```diff
@@ -1,2 +1,2 @@
 # Set of available languages
-es de fr ro ru
+es de fr ro
```
[the remark this answers is not in the document]
Comment (onkobu) [reply po/LINGUAS:2]: Don't know/ wasn't on purpose.

**Comment by printfn** [comment]
Any updates on this?

**Comment by onkobu** [comment]
> Any updates on this?

I am uncertain whether there is more work needed from my side regarding the missing ru…update my fork and in turn enhance this PR?

**Comment by onkobu** [comment]
Updated PR

**Pull request merged**